dags/dags1/jobs_dag_branching.py

```diff
@@ -17,24 +21,61 @@
 
 
 def print_process_start(**op_kwargs):
-    print('{} start processing tables in database: {}.'.format(op_kwargs['dag_id'], op_kwargs['database']))
+    print('{} start processing tables in database: {}.'.format(
+        op_kwargs.get('dag_id', None), op_kwargs.get('database', None)))
+
 
+def check_table_exist(sql_to_get_schema, sql_to_check_table_exist, table_name):
+    """ callable function to get schema name and after that check if table exist """
+    hook = PostgresHook('airflow_course_postgres')
+    # get schema name
+    query = hook.get_records(sql=sql_to_get_schema)
+    schema = None
+    for result in query:
+        if table_name in result:
+            schema = result[0]
+            print(schema)
+            break
 
-def check_table_exist():
-    if True:
+    # check table exist
+    query = hook.get_first(sql=sql_to_check_table_exist.format(schema, table_name))
+    print(query)
+    if query:
         return 'skip_table_creation'
     else:
         return 'create_table'
 
 
+def pull_user_value(context):
+    value = context['task_instance'].xcom_pull(task_ids='bash_get_user')
```

Comment:
both line could be combined like:
```python
return context['task_instance'].xcom_pull(task_ids='bash_get_user')
```
but I'm not sure is it good to create functions with one line.